src/eq_schema/Answer.js

```diff
@@ -46,7 +46,6 @@ class Answer {
       label,
       value: label
     };
```

Comment:
May as well revert this change, probably better to keep the blank line separating the `if` statement.